### DISCUSSION TITLE
Edit "Users" Table name

### DISCUSSION THREAD

**Discussion by Pachat** [Budibase/budibase#2667] [Help]
Hi,
Totally new to BB (1 hour) Creating my first App.
How do I edit the  "Users" Table name?

**Comment by mjashanks** [comment]
You can't! Sorry :)

Out of interest, why do you want to rename it?

**Comment by Pachat** [comment]
This is for non English speakers.
Is BB only in English?
What I may be did not catch is that it is only an internal name. In that case, effectively, it doesn't matter.

**Reply by mjashanks** [reply]
At the moment, Budibase is only in English. We hope to add multiple languages over time - it has been asked for a few times.

And yes, it's "Internal", i.e. is only inside the Builder, not the apps that your end users use. When you build your front-end, you can name things how you like.

**Reply by Pachat** [reply]
Great! Thanks!
